capacity4more/behat/features/group_status.feature

```diff
@@ -0,0 +1,91 @@
+Feature: Group Status
+  Test group status field
+
+  @api
+  Scenario: Check requested group as admin
+    Given I am logged in as user "mariecurie"
+    When  I start editing group "Requested group"
+    Then  the "#edit-c4m-og-status-und" element should not contain "archived"
+    And   the "#edit-c4m-og-status-und" element should not contain "published"
+
+  @api
+  Scenario: Check requested group as group owner
```

Comment:
@HelenaEksler A group owner has no access to its group until it is reviewed and approved.